Systems/Handbook/CollectibleBehaviorHandbookTextAndExtraInfo.cs

```diff
@@ -1289,6 +1291,11 @@ protected List<RichTextComponentBase> BuildBarrelRecipesText(ICoreClientAPI capi
                 ocmp.ShowStackSize = true;
                 barrelRecipesTexts.Add(ocmp);
 
+                string sealHoursText = (sealHours > 24.0) ? " " + Lang.Get("{0} days", Math.Round(sealHours / (double)capi.World.Calendar.HoursPerDay, 1)) : Lang.Get("{0} hours", Math.Round(sealHours));

```

Comment:
24.0 should be capi.World.Calendar.HoursPerDay as well so it properly adjusts if the setting is changed.

Reply:
> capi.World.Calendar.HoursPerDay

Done